src/config.js

```diff
@@ -38,8 +54,8 @@ function help() {
         --version, -v     show version
         --help, -h        show this
         
-        --add, -a         add alias
-        --remove, -r      remove alias
+        --add, -a         add alias (add -p for project alias)
```

Comment:
will cause many confuse with `--pull, -p   restore your config` (for me, at least)

maybe `-c` for 'cwd' or something?